libr/bin/format/elf/elf.c

```diff
@@ -784,7 +917,7 @@ static Sdb *store_versioninfo(struct Elf_(r_bin_elf_obj_t) *bin) {
 		}
 		if (bin->shdr[i].sh_offset + ((i + 1) * sizeof (Elf_(Shdr))) > bin->size) {
 			eprintf ("Warning: Too big version info field %d (%d)\n", i, size);
-			break;
+			//break;
```

Comment:
this break aims to speedup parsing 100x times when loading fuzzed bins that contain strangeous version information. you should see this in the times to run the whole testsuite